fedora_messaging/cli.py

```diff
@@ -87,7 +90,7 @@ def cli(conf):
 @click.option('--exchange', help=_exchange_help)
 @click.option('--amqp-url', help=_amqp_url_help)
 def consume(amqp_url, exchange, queue_name, routing_key, callback, app_name):
-
+    """Consume messages from an AMQP queue using a Python callback."""
```

Comment:
Might be nice to also document the params and their types, but this is still an improvement over what was there before so feel free to defer.

Reply:
I need to double check, but I believe click uses this docblock for user help, and it also renders the options help, so unfortunately it looks weird when you add the standard docblocks. I think I can add type hints to the click options, though.